data/mentors.js

```diff
@@ -0,0 +1,8 @@
+mentors = [
+	{
+		"name" : "Mentor 1",
```

Comment:
can't we extract all the mentor data from the projects? I.e. go through all projects, take their github handle, get the bio from github, get the avatar from github and the name?